x/autopilot/keeper/fallback.go

```diff
@@ -0,0 +1,39 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v16/x/autopilot/types"
+)
+
+// Stores a fallback address for an outbound transfer
+func (k Keeper) SetTransferFallbackAddress(ctx sdk.Context, channelId string, sequence uint64, address string) {
```

Comment:
What specifically do you find hard to review / did you mean this file specifically?

This wasn't copied over directly from anywhere but it follows the same format as all the other keepers in our codebase (across all modules and data structures) (examples: [host zone](https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/keeper/host_zone.go), [airdrop](https://github.com/Stride-Labs/stride/blob/main/x/claim/keeper/claim.go#L361), [icqs](https://github.com/Stride-Labs/stride/blob/main/x/interchainquery/keeper/queries.go#L83), [oracle](https://github.com/Stride-Labs/stride/blob/main/x/icaoracle/keeper/metric.go#L80), [ratelimit](https://github.com/Stride-Labs/stride/blob/main/x/ratelimit/keeper/rate_limit.go#L141), etc.)

Reply:
yeah I meant this file - I can diff it to the files you linked (although if you just copy + pasted from another keeper and changed variable names, it's probably not worth me diffing to other files)

Reply:
Yeah I wouldn't recommend diffing to those.

---

General recipe for keepers is:
```go
// set
  store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix({PREFIX_KEY_FOR_THIS_STRUCTURE))
  value := // some serialized version of the data you're storing
  key := // some serialized version of the key
  store.Set(key, value)

// get
  store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix({PREFIX_KEY_FOR_THIS_STRUCTURE))
  key := // some serialized version of the key
  valueBz := store.Get(key)
   if value not empty:
      // deserialize and return

// remove
  store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix({PREFIX_KEY_FOR_THIS_STRUCTURE))
  key := // some serialized version of the key
  store.Remove(key)
``` 

I wouldn't be too concerned with reviewing the boilerplate since we've done it a million times and there's unit tests. I'd just confirm things like:
  * Proper prefix key is used for substore
  * Proper key is used for structure
  * Serialization/deserialization is sound

Reply:
oh also, I should have mentioned that this is actually pretty identical to the rate limit `PendingSendPacket` keepers [[here](https://github.com/Stride-Labs/stride/blob/main/x/ratelimit/keeper/rate_limit.go#L191)] - same key with sequence + channel Id and same serialization

Reply:
I just reviewed against `PendingSendPacket` and against the recipe you provided. Both were very helpful.

Reply:
am I good to mark this resolved? @riley-stride @asalzmann

Reply:
(both said yes on slack) - resolving!